torch/csrc/jit/passes/quantization/quantization_patterns.h

```diff
@@ -1007,8 +1007,25 @@ graph(%packed_params, %a, %reduce_range, %a_dtype):
 graph(%packed_params, %a, %reduce_range, %a_dtype):
         %r = quantized::linear_dynamic(%a, %packed_params, %reduce_range)
         return (%r) )";
+
+  std::string linear_dynamic_fp16 = R"(
+graph(%packed_params, %a, %dtype_fp16, %dtype_fp32, %default_param, %non_blocking):
+        %fp16_tensor = aten::to(%a, %dtype_fp16, %default_param, %default_param, %non_blocking)
+        %fp32_tensor = aten::to(%fp16_tensor, %dtype_fp32, %default_param, %default_param, %non_blocking)
+        %w_unpacked : Tensor, %b : Tensor? = quantized::linear_unpack_fp16(%packed_params)
+        %r = aten::linear(%fp32_tensor, %w_unpacked, %b)
+        return (%r) )";
+
+  std::string quantized_linear_dynamic_fp16 = R"(
+graph(%packed_params, %a, %dtype_fp16, %dtype_fp32, %default_param, %non_blocking):
+        %r = quantized::linear_dynamic_fp16(%a, %packed_params)
+        return (%r) )";
+
   return {
       {"quantized::linear_dynamic", linear_dynamic, quantized_linear_dynamic},
+      {"quantized::linear_dynamic_fp16",
```

Comment:
same here

Reply:
maybe it's fine to not check here since it's checked before

Reply:
Done